src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
+      let userName = $('.username.input').val();
```

Comment:
`userName`'s value does not change, therefore we should use `const` instead of `let`. As this binding should not change during callback's execution, using `const` will tell others, that it really shouldn't change, and if for some reason `userName`'s value will change - we will get meaningful error.